src/test_utils.rs

```diff
@@ -366,6 +366,11 @@ pub fn number(n: &str) -> Value {
     Value::Number(n.parse().unwrap(), false)
 }
 
+/// Creates a `Value::SingleQuotedString`
+pub fn single_quoted_string(s: &str) -> Value {
+    Value::SingleQuotedString(s.to_string())
+}
```

Comment:
```suggestion
/// Creates a [Value::SingleQuotedString]
pub fn single_quoted_string(s: impl Into<String>) -> Value {
    Value::SingleQuotedString(s.into())
}
```
Thinking we can make this accept the various string types?